src/main/python/rlbot/gui/presets.py

```diff
@@ -0,0 +1,100 @@
+import os
+from PyQt5.QtCore import QTimer
+import configparser
+
+from rlbot.agents.base_agent import BaseAgent
+from rlbot.agents.base_agent import PYTHON_FILE_KEY, BOT_CONFIG_MODULE_HEADER
+from rlbot.utils.class_importer import import_agent, get_python_root
+
+
+class Preset:
```

Comment:
This should be the same name as the file name (probably rename the file in this case)
And add documentation

Reply:
Can we make a presets folder